.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -23,8 +23,8 @@ env:
   ASDF_MAGE_VERSION: 1.15.0
 
   # Integration Tests
-  K8S_VERSION: "v1.31.0"
-  ASDF_KIND_VERSION: "0.24.0"
+  K8S_VERSION: "v1.32.0"
+  ASDF_KIND_VERSION: "0.27.0"
```

Comment:
Wonder if these env variables could be centralised in the `pre-command` hook, in case they are the same values in every pipeline?

Reply:
Agree, I also prefer to keep the tool versions centralized in a single place. If you don't mind, I can do that as a follow up.